app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -45,9 +45,12 @@ def check_sp_handoff_bounced
 
     def confirm_user_is_authenticated_with_fresh_mfa
       bump_auth_count unless user_fully_authenticated?
-      return confirm_two_factor_authenticated(request_id) unless user_fully_authenticated? &&
-                                                                 service_provider_mfa_policy.
-                                                                 auth_method_confirms_to_sp_request?
+
+      unless user_fully_authenticated? && service_provider_mfa_policy.
+          auth_method_confirms_to_sp_request?
+        return confirm_two_factor_authenticated(request_id)
+      end
```

Comment:
This is what rubocop made me do. I'm not sure I love this; the `unless` continuation is further indented than the block contents.

The good news is that the method looks less like a map of Oklahoma.